static/js/dashboard.js

```diff
@@ -618,14 +625,22 @@ $(document).ready(function () {
           return `<span data-toggle="tooltip" data-placement="top" title="${data[0]} executed / ${data[1]} missed"><span class="text-success">${data[0]}</span> / <span class="text-danger">${data[1]}</span></span>`
         },
       },
+      // Performance last 7d
       {
-        // hidden column for filtering by DepositAddress
         targets: 9,
+        data: "9",
+        render: function (data, type, row, meta) {
+          return data
+        },
+      },
+      // Deposit address
+      {
+        targets: 10,
```

Comment:
`data: "9"` stands for the income and there was no code block for that before as the content for data[9] could be used as it is. Adding the block ranging from line 628 to 635 does not really do anything besides making everything more obvious.

Reply:
paprika